review/static/html/review_content_vertical.html

```diff
@@ -0,0 +1,9 @@
+<div>
+  <button class="review-button" aria-expanded="false" aria-controls="content-1" data-iframe-src="{VERTICAL_URL}">Review Problems</button>
+  <div id="content-1" class="review-content" aria-labelledby="section-heading-1">
```

Comment:
Should this id be hard-coded with ```1```?

Reply:
I think that should be fine because that is only when we are showing a vertical so we only ever display one at a time. Other parts of the code would have to change if we wanted to show multiple verticals and I feel like that could be handled then, if ever.